portfolio/src/main/webapp/index.html

```diff
@@ -6,6 +6,10 @@
 
     <title>Murilo Silva</title>
 
+    <!-- Fixes favicon.ico error -->
+    <link rel="shortcut icon" href="">
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBv7O45aUH5YEHh5qaNNd-_PSssUnvxW4M"></script>
```

Comment:
I've updated this key to restrict it to just the maps API, to prevent other uses (e.g. create VMs, etc.) and added a restriction for *.appspot.com

https://console.cloud.google.com/apis/credentials/key/ef0aec83-94e8-4c80-99c6-9a9e746a939c?folder=&organizationId=&project=msilva-sps-spring20